[FW][FIX] account_payment_pro_currency: recompute aml if change counterpart rate changes

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/account-payment#656

cav-adhoc committed:
[FIX] account_payment_pro_currency: recompute aml if change counterpart rate changes

X-original-commit: 07648896c691455cb15a02f8f809fc7f4aa671e2

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/657).

fw-bot-adhoc commented:
@cav-adhoc @maq-adhoc cherrypicking of pull request ingadhoc/account-payment#656 failed.

stdout:
```
CONFLICT (modify/delete): account_payment_pro_currency/views/account_payment_view.xml deleted in HEAD and modified in 69e6d514 ([FIX] account_payment_pro_currency: recompute aml if change counterpart rate changes). Version 69e6d514 ([FIX] account_payment_pro_currency: recompute aml if change counterpart rate changes) of account_payment_pro_currency/views/account_payment_view.xml left in tree.
CONFLICT (modify/delete): account_payment_pro_currency/models/account_payment.py deleted in HEAD and modified in 69e6d514 ([FIX] account_payment_pro_currency: recompute aml if change counterpart rate changes). Version 69e6d514 ([FIX] account_payment_pro_currency: recompute aml if change counterpart rate changes) of account_payment_pro_currency/models/account_payment.py left in tree.
CONFLICT (modify/delete): account_payment_pro_currency/__manifest__.py deleted in HEAD and modified in 69e6d514 ([FIX] account_payment_pro_currency: recompute aml if change counterpart rate changes). Version 69e6d514 ([FIX] account_payment_pro_currency: recompute aml if change counterpart rate changes) of account_payment_pro_currency/__manifest__.py left in tree.

```

stderr:
```
20:59:59.050999 git.c:444               trace: built-in: git cherry-pick 69e6d5144826547eddd0a9dd793adb996dfc8f13
error: could not apply 69e6d514... [FIX] account_payment_pro_currency: recompute aml if change counterpart rate changes
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port

cav-adhoc commented:
No es necesario en v18
